BE/src/main/resources/schema.sql

```diff
@@ -0,0 +1,86 @@
+DROP TABLE IF EXISTS member;
+CREATE TABLE member
+(
+    member_id BIGINT AUTO_INCREMENT NOT NULL,
+    email     VARCHAR(255) NOT NULL,
+    password  VARCHAR(255) NOT NULL,
+    PRIMARY KEY (member_id)
+);
+
+DROP TABLE IF EXISTS order_sheet;
+CREATE TABLE order_sheet
+(
+    order_id    BIGINT AUTO_INCREMENT NOT NULL,
```

Comment:
p1; 테이블명과 아이디를 일관성있게 일치시키는게 좋겠습니다. order_sheet 보다는 order가 좀 더 나아보이네요.
order가 예약어이므로 orders로 가도 괜찮을 듯 합니다. (또는 order_sheet_id로 가도 무방합니다.)

```suggestion
CREATE TABLE orders
(
    order_id    BIGINT AUTO_INCREMENT NOT NULL,
```

Reply:
아 넵 맞습니다. orders 로 수정하였습니다. 🙆‍♂️